src/components/commons/FormComponents/RenderInput.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable jsx-a11y/label-has-for */
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import './index.scss';
+
+
+class RenderInput extends Component {
+  static propTypes = {
+    id: PropTypes.string.isRequired,
+    value: PropTypes.oneOfType([PropTypes.string, PropTypes.bool, PropTypes.number]),
+    label: PropTypes.string.isRequired,
+    name: PropTypes.string.isRequired,
+    type: PropTypes.string.isRequired,
+    className: PropTypes.string.isRequired,
+    error: PropTypes.string,
+    handleChange: PropTypes.func.isRequired,
+  };
+
+  static defaultProps = {
+    value: '',
+    error: '',
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      changed: false
+    };
+  }
+
+  onFocus = () => {
+    this.setState({
+      changed: true
+    });
+  }
+
+  render() {
+    const {
+      id, type, name, className,
+      value, error, label, handleChange
+    } = this.props;
+    const { changed } = this.state;
+
+    return (
+      <Fragment>
+        <div className="my-4 relative mb-8">
+          <label htmlFor={id} className={classNames({ 'has-value': changed })}>{label}</label>
+
+          <input
+            id={id}
+            type={type}
+            name={name}
+            className={`${className} w-full py-2 border-gray-600 border-b-2 outline-none`}
+            value={value}
+            onChange={e => handleChange(e)}
```

Comment:
Don't do this.

do this instead
```suggestion
            onChange={handleChange}
```